src/dsverifier.cpp

```diff
@@ -1636,23 +1682,35 @@ void generates_mag_response(double* num, int lnum, double* den, int lden, double
 		out_numIm[i] = 0;
 		for (m = 1; m < lnum; ++m) {
 			old_out_Re = out_numRe[i];
-			out_numRe[i] = cos(w) * out_numRe[i] - sin(w) * out_numIm[i] + num[m];
-			out_numIm[i] = sin(w) * old_out_Re + cos(w) * out_numIm[i];
+			out_numRe[i] = cosTyl(w, SINE_precision) * out_numRe[i] - sinTyl(w, SINE_precision) * out_numIm[i] + num[m];
+			out_numIm[i] = sinTyl(w, SINE_precision) * old_out_Re + cosTyl(w, SINE_precision) * out_numIm[i];
 		}
 		out_denRe[i] = den[0];
 		out_denIm[i] = 0;
 
 		for (m = 1; m < lden; ++m) {
 			old_out_Re = out_denRe[i];
-			out_denRe[i] = cos(w) * out_denRe[i] - sin(w) * out_denIm[i] + den[m];
-			out_denIm[i] = sin(w) * old_out_Re + cos(w) * out_denIm[i];
+			out_denRe[i] = cosTyl(w, SINE_precision) * out_denRe[i] - sinTyl(w, SINE_precision) * out_denIm[i] + den[m];
+			out_denIm[i] = sinTyl(w, SINE_precision) * old_out_Re + cosTyl(w, SINE_precision) * out_denIm[i];
 		}
 
 		res[i] = sqrt(out_numRe[i] * out_numRe[i] + out_numIm[i] * out_numIm[i]); 
 	    zero_test = sqrt(out_denRe[i] * out_denRe[i] + out_denIm[i] * out_denIm[i]);
-	 	//if (zero_test != 0)
-		res[i] = res[i] / zero_test;
+		res[i] = res[i] /zero_test;
 	}
+
```

Comment:
I've removed the commented code, even thought I still think it's useful to just leave it there. As I stated in the comment, those commented lines were meant to be "decommented" in case of debugging purposes.